### PR TITLE
6.3.0 Pre Release

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,20 @@
...
-## [6.0.0](https://github.com/puppetlabs/puppetlabs-mysql/tree/6.0.0) (2018-07-19)
+## [6.1.0](https://github.com/puppetlabs/puppetlabs-mysql/tree/6.1.0) (2018-09-12)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-mysql/compare/6.0.0...6.1.0)
+
+### Added
+
+- pdksync - \(MODULES-7705\) - Bumping stdlib dependency from \< 5.0.0 to \< 6.0.0 [\#1114](https://github.com/puppetlabs/puppetlabs-mysql/pull/1114) ([pmcmaw](https://github.com/pmcmaw))
+
+### UNCATEGORIZED PRS; GO LABEL THEM
```
Comment (HelenCampbell) [inline CHANGELOG.md]: Fix this please.

Comment (david22swan) [reply CHANGELOG.md]: God damn am I blind

File: Rakefile [hunk 2]
```diff
@@ -6,7 +6,7 @@ require 'puppet_pot_generator/rake_tasks'
 
 def changelog_user
   return unless Rake.application.top_level_tasks.include? "changelog"
-  returnVal = nil || JSON.load(File.read('metadata.json'))['author']
+  returnVal = 'puppetlabs' || JSON.load(File.read('metadata.json'))['author']
```
Comment (pmcmaw) [inline Rakefile]: Where does this come from?

**Comment by pmcmaw** [comment]
Closing in favour for: https://github.com/puppetlabs/puppetlabs-mysql/pull/1120
Version is incorrect.

**Pull request closed without merging**